04-built-in-resource-types/Deployment/Deployment.nginx-better.yaml

```diff
@@ -28,6 +28,7 @@ spec:
           resources:
             limits:
               memory: "50Mi"
+              cpu: "250m"
```

Comment:
Mind removing the these limits?

https://home.robusta.dev/blog/stop-using-cpu-limits

Reply:
Removed